tests/eval-one.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+set -u
+here=$(cd "$(dirname "$0")" && pwd)
+
+if [ -z "$1" ]; then
+  echo "Usage: $0 <profile>"
+  exit 1
+fi
+
+profile=$1
+logfile=$(mktemp)
+
+# shellcheck disable=SC2039
```

Comment:
What's the point of using a tool when it requires magic comments to do its job? I think `shellcheck` is very overreaching, and we shouldn't use it.

Reply:
It catches a lot of common mistakes and educates the users about shell corner-cases.

Reply:
I don't think we need that for fifteen lines of shell script.